### ISSUE TITLE
.Nettrace traces lack some capabilities out of the box that the converted etlx files have

### ISSUE THREAD

**Issue by mrsharm** [microsoft/perfview#2026]
Seems like ETLX conversion of the nettrace file has features such as the Heap Analyzer, TraceInfo etc. These will be nice to have defaults.

## Nettrace

![image](https://github.com/microsoft/perfview/assets/68247673/b36a7b7a-d10c-4d6d-8e83-93be0b9dc38d)

## ETLX 

![image](https://github.com/microsoft/perfview/assets/68247673/ed481cc7-8039-4778-a879-f040795f212c)

**Comment by brianrob** [comment]
Yes, there are definitely some differences. TraceInfo isn't one that we can easily implement because nettrace files don't contain the necessary data. Same with Module Version. 

I recently implemented Processes for multi-process nettrace support. 

Happy to add GC Heap Analyzer. This is done in PerfViewData.cs by adding the view in EventPipePerfViewFile. There is an example of how to do this in ETLPerfViewFile. All of this is in PerfViewData.cs.

Let's see if @copilot can do this.
